### ISSUE TITLE
struct.decode with empty listValue raises "Cannot read property 'map' of undefined"

### ISSUE THREAD

**Issue by fontelemos** [callmehiphop/pb-util#15]
Hello :D !
I've had an issue running `struct.decode` with empty lists generated by the same Python lib mentioned in #13 .
Here's an example:
```json
{ "fields": { "test": { "listValue": {} } } }
```

It raises the following exception at line https://github.com/callmehiphop/pb-util/blob/master/index.ts#L209
```
Cannot read property 'map' of undefined
```
I've tested changing the line https://github.com/callmehiphop/pb-util/blob/master/index.ts#L208 of `pb-util`  to the following and it seems to resolve the issue:
```ts
decode({values = []}: ListValue): JsonArray {
```

**Comment by callmehiphop** [comment]
Thanks for the report, @Leinvedan! The fix looks good to me and I'd be happy to accept a PR if that's something you're interested in, otherwise I can get this done pronto.

**Comment by fontelemos** [comment]
Sure! I'll create the PR and send it right away :)

**Comment by callmehiphop** [comment]
I just published a patch release (v1.0.3), so this should be resolved now. Let me know if the issue still persists somehow and thanks again for the contribution!
